portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -1,4 +1,3 @@
-// Copyright 2019 Google LLC
```

Comment:
Please make sure you run the formatter on the file.

Reply:
Updated!